docs/faq/help-center.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I'm not sure it is necessary to change the file name (which impacts the URL).

If you do change a page URL, I highly recommend to add a redirect:

https://github.com/lukso-network/docs/blob/main/docusaurus.config.js#L19

This is for SEO purposes and also to make sure all exising links to this page don't go to a 404.

But i think we can keep "help.md" (so `/help` URL)

Reply:
Agree on this